albatross.ml

```diff
@@ -401,16 +401,8 @@ module Make (S : Tcpip.Stack.V4V6) = struct
 
   let console name tls_flow d =
     let open Lwt.Infix in
-    Lwt.choose
```

Comment:
I think here you wanted Lwt.pick over Lwt.choose so the timeout is cancelled and doesn't close the flow.